App.tsx

```diff
@@ -167,14 +169,14 @@ const BaseApp: React.FunctionComponent<{
                   }
                 },
               })}>
-              <TabNavigator.Screen name="Home" initialParams={{center_id: avalancheCenterId, dateString: dateString}}>
-                {state => HomeTabScreen(withParams(state, {center_id: avalancheCenterId, dateString: dateString}))}
+              <TabNavigator.Screen name="Home" initialParams={{center_id: avalancheCenterId, dateString}}>
+                {state => HomeTabScreen(merge(state, {route: {params: {center_id: avalancheCenterId, dateString}}}))}
```

Comment:
The compiler was convinced that with `withParams`, we were losing the `screen` field of `params` (which comes in via `NavigationScreenParams`). `merge` is designed for this job anyway and seems to not offend tsc.